From fe0f05384d21932c21bfa9300a6e473367254139 Mon Sep 17 00:00:00 2001
From: Tobias Schlemmer <keinstein@users.sourceforge.net>
Date: Sat, 22 Sep 2018 21:32:32 +0200
Subject: Enhance .travis.yml

* Explicitely set distribution and derivative. This allows to build tags.
* package debian.tar.gz
---
 .travis.yml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/.travis.yml b/.travis.yml
index d555fa54..0ec7c6a9 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -1,10 +1,15 @@
 sudo: required
 language: generic
 
+env:
+    - TRAVIS_DEBIAN_DERIVATIVE=debian
+      TRAVIS_DEBIAN_DISTRIBUTION=stretch
+
 services:
   - docker
 
 script:
+    - env
     - wget -O- http://travis.debian.net/script.sh | sh -
     - ls ..
 
@@ -18,6 +23,8 @@ deploy:
     - ../*.deb
     - ../*.dsc
     - ../*.changes
+    - ../*.tar.*
+
   on:
     repo: keinstein/mutabor
     tags: true
